.ci/run-e2e-tests.sh

```diff
@@ -39,6 +39,16 @@ elif [ "${TEST_GROUP}" = "examples2" ]
 then
     echo "Running Examples2 Tests"
     make e2e-tests-examples2
+elif [ "${TEST_GROUP}" = "es-token-propagation" ]
+then
+    echo "Running token propagation tests"
+    oc create user user-test-token
+    oc adm policy add-cluster-role-to-user cluster-admin user-test-token
+    # for ocp 4.2
+    htpasswd -c -B -b users.htpasswd user-test-token any
```

Comment:
It's a better idea to just ship the actual file instead of generating the file at the CI.